doc/devel/testing.rst

```diff
@@ -264,10 +264,10 @@ Using tox
 `Tox <https://tox.readthedocs.io/en/latest/>`_ is a tool for running
 tests against
 multiple Python environments, including multiple versions of Python
-(e.g., 2.7, 3.4, 3.5) and even different Python implementations
+(e.g., 3.5, 3.6) and even different Python implementations
```

Comment:
The `2.2.x` series still supports python 2, so I that maybe these bits should still have 2.7 in them.

Reply:
I looked at this when I did the backport, decided it was just prose to illustrate what 'multiple versions of python' meant and left it (because I was feeling lazy.....)